### ISSUE TITLE
Using Test2::Bundle::More in stepfiles makes tests "bleed through" on terminal

### ISSUE THREAD

**Issue by ehuelsmann** [pherkin/test-bdd-cucumber-perl#146]
@kwakwaversal reported on gitter:
> can you think of any reason why using the Test2::Suite of tools would enable verbosity in the output for both pherkin and prove? ( https://github.com/kwakwaversal/test-perl-cucumber )

```plain
$ pherkin t/cucumber/
1..0
  this is a test
    Oh and this is a description. Neat, huh?

    Scenario: Testing with Test2::Bundle::More
ok 1
      Given foo
ok 2
      Then bar

$ prove --source Feature --feature-option extensions=.feature --ext=.feature -r t/
1..0
t/cucumber/basic.feature .. ok 1
t/cucumber/basic.feature .. 1/? ok 2
t/cucumber/basic.feature .. ok
All tests successful.
Files=1, Tests=2,  0 wallclock secs ( 0.20 usr +  0.02 sys =  0.22 CPU)
Result: PASS
```

**Comment by ehuelsmann** [comment]
@kwakwaversal, an option might be to offer 2 versions of the module:

1. Test::BDD::Cucumber - which works with Test::More
2. Test2::BDD::Cucumber - which works with Test2::Suite

Would this be a solution for you? Or are you mixing Test::More and Test2::Bundle::More in a single project?

**Comment by ehuelsmann** [comment]
Progress on this issue: the `test2-api` branch works with both Test2::Bundle::More and Test::More, except that it doesn't for Test::More with `$TODO`. I'm working on that bit with Chad.

**Comment by ehuelsmann** [comment]
Resolved in 9046d51907da88e60bee731e08af09d6febda980
